packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -45,12 +45,7 @@ const highContrastItemAndTitleStateMixin: IRawStyle = {
     [HighContrastSelector]: {
       backgroundColor: 'Highlight',
       borderColor: 'Highlight',
-      color: 'HighlightText',
-      selectors: {
-        ':hover': {
-          color: 'HighlightText' // overrides the hover styling for buttons that are also selected
```

Comment:
removed since it doesn't seem to be needed. original change: https://github.com/OfficeDev/office-ui-fabric-react/issues/5238